tools/ldb_cmd.cc

```diff
@@ -312,23 +312,19 @@ LDBCommand::LDBCommand(const std::map<std::string, std::string>& options,
 
 void LDBCommand::OpenDB() {
   Options opt;
-  bool opt_set = false;
   if (!create_if_missing_ && try_load_options_) {
     Status s = LoadLatestOptions(db_path_, Env::Default(), &opt,
                                  &column_families_, ignore_unknown_options_);
-    if (s.ok()) {
-      opt_set = true;
-    } else if (!s.IsNotFound()) {
+    options_ = opt;
```

Comment:
can we merge `options_` and `opt`?

Reply:
Yes, it might've just been there as an abbreviation

Reply:
Also having PrepareOptionsForOpenDB return an Options object seems pretty pointless since it can just assign to options_. But I don't want to go too far into refactoring this file right now.